src/node/transaction.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <consensus/validation.h>
 #include <net.h>
```

Comment:
nit: you can remove the `net.h` include now that `CNode` is not used in this file.